views/index.erb

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Chitter Message Board</title>
+        <style>
+            body {
+                font-family: helvetica, sans-serif;
```

Comment:
Styling looks great in the browser, but would suggest moving all the CSS code to a separate stylesheet file so you can reuse it more easily as the application grows